vllm/sequence.py

```diff
@@ -139,7 +139,18 @@ def get_output_len(self) -> int:
         return len(self.output_token_ids)
 
     def get_token_ids(self) -> List[int]:
-        return self.prompt_token_ids + self.output_token_ids
+        return list(self.prompt_token_ids) + self.output_token_ids
```

Comment:
Agreed. For performance we need to avoid tuple -> list conversion. How about storing the list version as `prompt_token_ids` for accessing, and stores the tuple version in `_prompt_token_ids_tuple` for hashing speedup purposes?

Reply:
yeah I think that sounds good to me. (it should be fine since prompt tokens are not going to be changed)

Reply:
Revert this part?